src/syrupy/assertion.py

```diff
@@ -88,10 +89,13 @@ def use_extension(
     def assert_match(self, data: "SerializableData") -> None:
         assert self == data
 
+    def _serialize(self, data: "SerializableData") -> "SerializedData":
+        return self.extension.serialize(data, matcher=self._matcher)
```

Comment:
Might be a breaking change if extension serializers were not written with support for variable keyword arguments i.e. `**kwargs`.

Reply:
Worth forcing it with *?

Reply:
> Since the serializer is a public API, yes I believe it's a breaking change. How do we prevent this in the future? A note to use **kwargs? What do other python projects do?

@noahnu we could force the use of kwargs in the serializer?

```py
def serialize(self, *, **kwargs)...
```